### PR TITLE
Generate session secret if blank

### PR DESCRIPTION
**Pull request by parente**
Don't silently use a fixed, insecure default

### CODE REVIEW

**Comment by parente** [comment]
@jhpedemonte can you give this a quick look-see?

File: app/config.js [hunk 1]
```diff
@@ -75,4 +76,12 @@ if (key_file_location || cert_file_location) {
     });
 }
 
+// Generate a session key if one is not specified. Log that we did it in case
+// the user accidentally left of an explict token.
```
Comment (jhpedemonte) [inline app/config.js]: of => off

**Comment by jhpedemonte** [comment]
Looks good, only saw a minor misspelling. Fix that and merge, I say!

File: app/config.js (hunk 1 above)
Comment (parente) [inline app/config.js]: All set.

**Pull request merged**